### PR TITLE
Fix parameter -MinerName & -ExludeMinerName

### PR DESCRIPTION
**Pull request by UselessGuru**
Current dual miners do not work with -MinerName or -ExcludeMinerName
Solution: insert a dash between miner name and algorithm name.

### CODE REVIEW

File: MinersLegacy/Excavator.ps1 [hunk 1]
```diff
@@ -43,7 +43,7 @@ $Name = Get-Item $MyInvocation.MyCommand.Path | Select-Object -ExpandProperty Ba
 $Devices = $Devices | Where-Object Type -EQ "GPU" | Where-Object Vendor -EQ "NVIDIA Corporation"
 
 $Devices | Select-Object Vendor, Model -Unique | ForEach-Object {
-    $Miner_Device = $Devices | Where-Object Vendor -EQ $_.Vendor | Where-Object Model -EQ $_.Model
+    $Miner_Device = $Devices | Where-Object Model -EQ $_.Model
```
Comment (UselessGuru) [inline MinersLegacy/Excavator.ps1]: Where-Object Vendor -EQ $_.Vendor |
is redundant - is already in line 43

**Comment by grantemsley** [comment]
Would it be easy to make it work both ways?  So I can specify the first part of the miner name to exclude everything from that miner, or more of it to disable just a particular one?

So for ClayMoreEthash:

`ClaymoreEthash` - excludes the miner entirely
`ClaymoreEthash-Keccak` exclues Keccak secondary algorithm
`ClaymoreEthash-Keccak-40` excludes only the 40 intensity for keccak
`ClaymoreEthash-Keccak-40-GPU#00` excludes the 40 intensity for keccak on GPU 0 only.

Should be easy to do by matching whatever is specified with a dash appended.  Probably best if it has to end just before a dash, so it doesn't match other miners by accident.

The reason I ask is that there are a couple specific algorithms I'd love to disable only on my primary GPU - some are fine, others make the system so laggy I can't move the mouse properly.

**Comment by UselessGuru** [comment]
> Would it be easy to make it work both ways? 

Don't think so. Compare-Object does not allow wildcards :-(

File: MinersLegacy/Excavator.ps1 (hunk 1 above)
Comment (aaronsace) [reply MinersLegacy/Excavator.ps1]: Lets keep it in as to not transpose an error to other miners based off this.

**Pull request merged**